src/main/java/org/apache/commons/imaging/formats/webp/chunks/WebPChunkVP8X.java

```diff
@@ -0,0 +1,118 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.imaging.formats.webp.chunks;
+
+import org.apache.commons.imaging.ImagingException;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+/**
+ * <pre>{@code
+ *  0                   1                   2                   3
+ *  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |                                                               |
+ * |                   WebP file header (12 bytes)                 |
+ * |                                                               |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |                      ChunkHeader('VP8X')                      |
+ * |                                                               |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |Rsv|I|L|E|X|A|R|                   Reserved                    |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * |          Canvas Width Minus One               |             ...
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * ...  Canvas Height Minus One    |
+ * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * }</pre>
+ *
+ * @see <a href="https://developers.google.com/speed/webp/docs/riff_container#extended_file_format">Extended File Format</a>
+ *
+ * @since 1.0-alpha4
+ */
+public final class WebPChunkVP8X extends WebPChunk {
+    private final boolean hasICC;
+    private final boolean hasAlpha;
+    private final boolean hasExif;
+    private final boolean hasXmp;
+    private final boolean isAnimation;
+    private final int canvasWidth;
+    private final int canvasHeight;
+
+    public WebPChunkVP8X(int type, int size, byte[] bytes) throws ImagingException {
+        super(type, size, bytes);
+
+        if (size != 10) {
+            throw new ImagingException("VP8X chunk size must be 10");
+        }
+
+        int mark = bytes[0] & 0xFF;
+        this.hasICC = (mark & 0b0010_0000) != 0;
+        this.hasAlpha = (mark & 0b0001_0000) != 0;
+        this.hasExif = (mark & 0b0000_1000) != 0;
+        this.hasXmp = (mark & 0b0000_0100) != 0;
+        this.isAnimation = (mark & 0b0000_0010) != 0;
+
+        this.canvasWidth = (bytes[4] & 0xFF) + ((bytes[5] & 0xFF) << 8) + ((bytes[6] & 0xFF) << 16) + 1;
+        this.canvasHeight = (bytes[7] & 0xFF) + ((bytes[8] & 0xFF) << 8) + ((bytes[9] & 0xFF) << 16) + 1;
+
+        if (canvasWidth < 0 || canvasHeight < 0 || canvasWidth * canvasHeight < 0) {
+            throw new ImagingException("Illegal canvas size");
+        }
+    }
+
+    public boolean isHasICC() {
+        return hasICC;
+    }
+
+    public boolean isHasAlpha() {
```

Comment:
@Glavo would it OK to call this method "hasAlpha()" instead of "isHasAlpha" ? Ditto for the other methods? (I can batch rename them as I'm modifying the code, writing Javadocs, etc)

Reply:
Done! Maintaining the variable as `hasAlpha`.